frontend/pages/docs/index.vue

```diff
@@ -3,12 +3,11 @@
     class="text-light-text dark:text-dark-text bg-light-content dark:bg-dark-content"
   >
     <Head>
-      <Title>{{ title }}</Title>
+      <Title>{{ $t("pages.docs.index.title") }}</Title>
```

Comment:
Btw, I really like how you also went through and are doing the localization on a deep level like for the page header and such. This is so great :) :)